### DISCUSSION TITLE
amamba/user-guide/gitops/argo-cd-status/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1777] [Announcements]
# amamba/user-guide/gitops/argo-cd-status/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/user-guide/gitops/argo-cd-status/

**Comment by liyuerich** [comment]
健康状态能解释一下吗？ 从描述中还不太能理解具体含义。 或者什么情况下会被降级等
